lib/msal-node/src/cache/CacheContext.ts

```diff
@@ -26,54 +22,25 @@ export class CacheContext {
         AppMetadata: {}
     };
 
-    constructor() {
-        this.cachePath = "";
-    }
-
-    /**
-     * sets the cache path provided by the user
-     * @param path
-     */
-    setCachePath(path: string) {
-        this.cachePath = path;
-    }
+    constructor() {}
```

Comment:
Is this needed?

Reply:
if you are removing `cachePath`, we no longer need a constructor IMO.

Reply:
Empty constructor is not needed.